smtsolver.js

```diff
@@ -53,5 +53,6 @@ function solve (query) {
 }
 
 module.exports = {
-  smtSolver: solve
+  smtSolver: solve,
+  availableSolvers: solvers
```

Comment:
Why is this needed? And if it is needed, would it make sense mapping it into names only?

Reply:
It is needed by the `smtcallback.js` test such that it can decide not to run the tests if there's no solver installed. Names only should be fine. Even just the amount would be fine

Reply:
Updated